protocol/src/protocol.fbs

```diff
@@ -65,28 +69,28 @@ table Transaction {
 }
 
 table OutPoint {
-    hash:           Bytes;
+    hash:           H256;
     index:          uint32;
 }
 
 table CellInput {
-    hash:           Bytes;
+    hash:           H256;
     index:          uint32;
     unlock:         Script;
 }
 
 table CellOutput {
     capacity:       uint64;
-    data:           Bytes;
-    lock:           Bytes;
+    data:           [ubyte];
+    lock:           H256;
     contract:       Script;
 }
 
 table Script {
     version:        uint8;
     args:           [Bytes];
```

Comment:
If there isn't a way to use something like `[[ubyte]]`, I suggest we still keep `Bytes` instead of `[ubyte]` for consistency.

Reply:
regarding `Bytes`, if we change `short_ids ` to a new  [u8; 10] struct `ShortId` like `H256`, we can rename it to 'Arg', it's only used in `args` and `signed_args`

Reply:
Let's keep Bytes instead of `[ubyte]` for now.